rollbar/lib/transforms/scruburl.py

```diff
@@ -24,12 +24,7 @@ def __init__(self,
         self.params_to_scrub = set(map(lambda x: x.lower(), params_to_scrub))
 
     def in_scrub_fields(self, key):
-        if not key:
-            # This can happen if the transform is applied to a non-object,
-            # like a string.
-            return True
-
-        return super(ScrubUrlTransform, self).in_scrub_fields(key)
+        return True
```

Comment:
It's on purpose. The reason is that we want to scrub URLs out of every string, not just ones that we know the key for.